coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 98.34
+    "line": 98.99
```

Comment:
"line"? That's new, is it supposed to be like that?

Reply:
Yes, SimpleCov 0.18 (released in January, 2020) added "branch coverage". The type of coverage we _have_ been using is "line coverage" they renamed it to avoid ambiguity. The test reporter gem was holding back our version of SimpleCov which is why this change showed up here.